verification/src/http_server/handlers/verification/solidity/sourcify.rs

```diff
@@ -0,0 +1,70 @@
+use actix_web::{error, error::Error, web::Json};
```

Comment:
Not quite sure that I get you. In general all those `Sourcify*` structures (except `SourcifyRequest`) are not exposed outside the module. So, we can name them as would be clearer for the `sourcify` mod. But I didn't get what structures do you mean

Reply:
I'm talking about `SourcifyRequest`, `SourcifyVerifyResponse`, `sourcify_verification_request`

> So, we can name them as would be clearer for the sourcify mod

Isn't it will be clearer, if we'd have just `Request`, `VerifyResponse` and `verification_request` inside sourcify? This is a sourcify module and these are local to sourcify module types and functions, so we don't need to add the prefix to the names

This suggestion comes from [go](https://go.dev/doc/effective_go#package-names), where they suggest use package names as prefixes to the names:

> The importer of a package will use the name to refer to its contents, so exported names in the package can use that fact to avoid repetition. <...> For instance, the buffered reader type in the bufio package is called `Reader`, not `BufReader`, because users see it as `bufio.Reader`, which is a clear, concise name. Moreover, because imported entities are always addressed with their package name, `bufio.Reader` does not conflict with `io.Reader`. <...> Use the package structure to help you choose good names.

In Rust we don't necessary need to use package names, but we can, and I think in this example it can be easier, if we move prefix to the module name

Reply:
agree with lev, think its better to remove those sourcify prefixes